src/components/PWAInstructions/PWAInstructions.module.scss

```diff
@@ -28,27 +28,27 @@
           margin-bottom: 1rem;
           &_toggles {
             :global(.MuiToggleButtonGroup-grouped:not(:first-child)) {
-              border-left: 1px solid $neutral-white-opacity50;
+              border-left: 1px solid var(--mui-palette-neutral-light);
```

Comment:
Did you double check that this looked alright, just wanting to make sure since the color originally had some opacity to it.

Reply:
This was the border between the buttons for the 360 install instructions page, but changing the color to the regular white color seemed good, it didn't change how it looks and you can barely tell.